packages-ui/roosterjs-react/lib/contextMenu/menus/createImageEditMenuProvider.tsx

```diff
@@ -82,6 +82,41 @@ const ImageResizeMenuItem: ContextMenuItem<ImageEditMenuItemStringKey, ImageEdit
     },
 };
 
+const ImageRotateMenuItem: ContextMenuItem<ImageEditMenuItemStringKey, ImageEdit> = {
+    key: 'menuNameImageRotate',
+    unlocalizedText: 'Rotate image',
+    subItems: {
+        menuNameImageRotateLeft: 'Left',
+        menuNameImageRotateRight: 'Right',
+        menuNameImageRotateUpsidedown: 'Flip vertically',
```

Comment:
Shouldn't this be something like "Rotate 180 degree"?

I tried it and I got 
<img width="318" alt="image" src="https://user-images.githubusercontent.com/23065085/224441750-1680b161-a9d2-4b3c-9c02-acfb8c302472.png">

I think for "flip vertically" it means show a up side down mirrored image, but not rotated.